main.tf

```diff
@@ -0,0 +1,112 @@
+# Create a GCS bucket to store our critical application state into.
+module "gcs" {
+  source     = "./modules/gcs"
```

Comment:
do we want to have this be not the acronym? 

is `gcs` easier to parse over something like `cloud_storage`?

Reply:
Good call. I need to do that a few other places.